index.android.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var React = require('react-native');
+var {
+  AppRegistry,
+  StyleSheet,
+  Component,
+  Text,
+  View,
+  Navigator,
+  TouchableOpacity,
+} = React;
+
+var LoginScreen = require('./LoginScreen');
+var MainPage    = require('./MainPage');
+
+class AwesomeProject extends Component {
+  
+  render() {
+    return (
+
+       <Navigator
```

Comment:
Navigation/Router is a really hard problem, even harder in Mobile.

A Login flow as simple as it seems it is not so simple in fact.

You need redux to store the login information (you should watch this great video - https://www.youtube.com/watch?v=d7cH3VKw_Yo)

And all the videos of https://egghead.io/series/getting-started-with-redux of @gaeron 

The Navigator API has a lot of problems, to be easier to work with this, you should consider using this lib - https://github.com/aksonov/react-native-router-flux

Login is 'hard' because it needs an async action dispatcher, that is not so simple for new comers.

I really recommend https://github.com/yelouafi/redux-saga to handle async actions

You can always ask help for me, @keuller and @henriquesosa on https://gitter.im/henriquesosa/react-native-dojo